emloop/hooks/save_masks.py

```diff
@@ -0,0 +1,67 @@
+"""
+Hook for saving masks to the filesystem.
+"""
+
+import os
+import collections
+import logging
+
+import cv2
```

Comment:
Please only `try` this import and if it fails, produce a n exception with a reasonable message informing the user that this is an optional and requires OpenCV. Make sure that emloop works even when opencv is not installed!

Reply:
@FloopCZ so the package & dependency installation in CircleCI config should stay there? And if the import produces exception than tests will fail so should I skip them in that case?

Reply:
Yes, basically
1. catch `ImportError` as @FloopCZ suggested
2. in one test-container: install opencv in CircleCI and run full tests (all must succeed)
3. in second test-container: don't install opencv and run all tests without those requiring opencv (all must succeed)